code/KotobaUITests/KotobaUITests.swift

```diff
@@ -120,9 +121,7 @@ class KotobaUITests: XCTestCase
 		let oneCell = table.cells.element
 		oneCell.tap()
 
-		sleep(1)
-
-		XCTAssert(app.buttons["Manage"].exists)
+		XCTAssertTrue(app.navigationBars.staticTexts["one"].waitForExistence(timeout: 3))
```

Comment:
I believe that the contents of the dictionary view are now out-of-process and are not accessible to the `XCUIApplication`. I changed this to look for something in the navigation bar.